model-engine/model_engine_server/core/domain_exceptions.py

```diff
@@ -57,3 +57,8 @@ class ReadOnlyDatabaseException(DomainException):
     """
     Thrown if the server attempted to write to a read-only database.
     """
+
+class InvalidRequestException(DomainException):
```

Comment:
it already exists in model_engine_server.domain.exceptions

Reply:
not sure difference between `model_engine_server.core.domain_exceptions` and `model_engine_server.domain.exceptions` though

Reply:
oh, sorry about that -- I'll import from `model_engine_server.domain.exceptions`, I can't think of a reason why we'd need this in two places

Reply:
Actually we should probably look into why we have two different sets of exceptions and combine them if possible. I'll look into that